upgrading envoy to 1.33.2 and fixing CVEs

nitin-sachdev-29 opened the pull request:
upgrading envoy to 1.33.2 and fixing CVEs GO-2025-3595 in golang.org/x/net@v0.37.0

nitin-sachdev-29 committed:
upgrading envoy to 1.33.2 and fixing CVEs GO-2025-3595 in golang.org/x/net@v0.37.0

nitin-sachdev-29 committed:
added changelog

sriramr98 commented:
LGTM

nitin-sachdev-29 committed:
executed go mod tidy

anandmukul93 reviewed (changes requested):
Changelog doesn't contain go - 1.23.8 migration

nitin-sachdev-29 committed:
fixed changelog

anandmukul93 reviewed (dismissed):
LGTM

nitin-sachdev-29 committed:
upgraded go version

nitin-sachdev-29 committed:
upgraded ubuntu to latest

nitin-sachdev-29 committed:
fixed build pipeline

nitin-sachdev-29 committed:
changed ubuntu version

anandmukul93 committed:
Update build.yml

24.04 failing due to glibc version not available. 
Falling back to 22.04 for testing

anandmukul93 committed:
Fixed glibc libraries not found with a step for installing them

shore reviewed (dismissed):
Workflow update looks good to me!

anandmukul93 committed:
build test fix

anandmukul93 committed:
Trigger Build

anandmukul93 committed:
clubbed container

anandmukul93 committed:
added local docker repository service

anandmukul93 committed:
directory failure fix

anandmukul93 committed:
bin_name passing to docker fix

anandmukul93 committed:
make clean optional

anandmukul93 committed:
reproducible nope setting test

anandmukul93 committed:
fixing ubuntu-runner for docker build step

anandmukul93 committed:
fix : CGO_ENABLED=0 for non-fips as env not set for actions-go-build

nitin-sachdev-29 committed:
chore: update CODEOWNERS to include consul-selfmanage-maintainers for release configuration

anandmukul93 reviewed (approved):
LGTM
